pom.xml

```diff
@@ -2,15 +2,15 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.hsl.transitdata</groupId>
     <artifactId>pulsar-mqtt-gateway</artifactId>
-    <version>0.1.4</version>
+    <version>0.1.5</version>
     <packaging>jar</packaging>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <pulsar.version>2.1.0-incubating</pulsar.version>
-        <common.version>0.2.3</common.version>
+        <common.version>0.7.4</common.version>
```

Comment:
because of the major version bump we should test it a bit more. specially error handling has been improved in versions 0.3.x and 0.4.x